src/Passes/Source/Llvm/Llvm.hpp

```diff
@@ -41,11 +41,22 @@
 #include "llvm/IR/Function.h"
 #include "llvm/IR/IRBuilder.h"
 #include "llvm/IR/LLVMContext.h"
-#include "llvm/IR/LegacyPassManager.h"
 #include "llvm/IR/Module.h"
 #include "llvm/IR/Type.h"
 #include "llvm/IR/Verifier.h"
 
+// Reader tool
+#include "llvm/IRReader/IRReader.h"
+#include "llvm/Support/SourceMgr.h"
+#include "llvm/Transforms/IPO/Inliner.h"
+#include "llvm/Transforms/Scalar/LoopUnrollPass.h"
+
+// Profiles
+#include "llvm/IR/LegacyPassManager.h"
```

Comment:
Are the passes currently written using the new pass manager, the legacy one, or both?

Reply:
We only use the new one and I believe we can remove this line.